.github/workflows/push-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
```

Comment:
Let's remove the release branch and tags triggers, and just do this on the master branch for now. In a later PR we can enable for the others after we are sure master is stable with this.

Reply:
done.